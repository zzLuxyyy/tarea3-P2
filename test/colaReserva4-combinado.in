crearColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearReserva
encolarColaReservas
frenteColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
crearReserva
encolarColaReservas
imprimirColaReservas
crearSocio 23456789 Lucia Perez 10 03 2010
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata parlante que reflexiona sobre la historia" 2
crearReserva
encolarColaReservas
cantidadColaReservas
crearSocio 34567890 Juan Diaz 10 03 2005
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
crearReserva
encolarColaReservas
cantidadColaReservas
imprimirColaReservas
desencolarColaReservas
imprimirColaReservas
frenteColaReservas
cantidadColaReservas
liberarColaReservas
Fin