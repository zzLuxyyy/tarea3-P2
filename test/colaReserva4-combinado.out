1>Se ejecutó 'crearTColaReservas' exitosamente
2>Socio de ci 12345678 creado exitosamente.
3>4>5>Reserva de libro de isbn 411222335 a socio 12345678 creada exitosamente.
6>Se ejecutó 'encolarTColaReservas' exitosamente.
7>Reserva obtenida al ejecutar frente:
Reserva de libro Todos los fuegos el fuego por Jim Dear.
8>Socio de ci 12345678 creado exitosamente.
9>10>11>Reserva de libro de isbn 431244557 a socio 12345678 creada exitosamente.
12>Se ejecutó 'encolarTColaReservas' exitosamente.
13>Cola de Reservas:
Reserva de libro Todos los fuegos el fuego por Jim Dear.
Reserva de libro Lo bello y lo triste por Jim Dear.
14>Socio de ci 23456789 creado exitosamente.
15>16>17>Reserva de libro de isbn 421233446 a socio 23456789 creada exitosamente.
18>Se ejecutó 'encolarTColaReservas' exitosamente.
19>La cantidad de reservas en la cola es 3
20>Socio de ci 34567890 creado exitosamente.
21>22>23>Reserva de libro de isbn 431244557 a socio 34567890 creada exitosamente.
24>Se ejecutó 'encolarTColaReservas' exitosamente.
25>La cantidad de reservas en la cola es 4
26>Cola de Reservas:
Reserva de libro Todos los fuegos el fuego por Jim Dear.
Reserva de libro Lo bello y lo triste por Jim Dear.
Reserva de libro La ratesa por Lucia Perez.
Reserva de libro Lo bello y lo triste por Juan Diaz.
27>Se ejecutó 'desencolarTColaReservas' exitosamente
28>Cola de Reservas:
Reserva de libro Lo bello y lo triste por Jim Dear.
Reserva de libro La ratesa por Lucia Perez.
Reserva de libro Lo bello y lo triste por Juan Diaz.
29>Reserva obtenida al ejecutar frente:
Reserva de libro Lo bello y lo triste por Jim Dear.
30>La cantidad de reservas en la cola es 3
31>Se ejecutó 'liberarTColaReservas' exitosamente
32>Fin.
